### PR TITLE
"About" page (with refactoring "How" page)

### PR DESCRIPTION
**Pull request by avine**

### CODE REVIEW

File: src/styles/_modal.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+  <article class="modal-container modal-container--with-space">
```
Comment (blacroix) [inline src/styles/_modal.scss]: Code may be deleted?

**Review by blacroix — APPROVED** [review]

File: src/styles/_modal.scss (hunk 1 above)
Comment (avine) [reply src/styles/_modal.scss]: This comment show the expected markup for this CSS rules.
Why not keeping it ?

File: src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        </p>
+        <p className="modal-paragraph">
+          Solution globale au sein du Groupe Publicis,
+          nos 300 Sapient France accompagnent localement
+          nos clients avec la force du réseau mondial de Sapient
+          et le Power of One Publicis.<br/>
+          <a href="https://www.publicissapient.fr" className="modal-link">www.publicissapient.fr</a>
+        </p>
+        <p className="modal-paragraph">
+          <Button isMini linkTo="https://blog.engineering.publicissapient.fr/">Visitez notre blog</Button>
```
Comment (blacroix) [inline src/pages/about.tsx:40]: We could use env variable for that maybe? And use https://www.gatsbyjs.com/plugins/gatsby-plugin-google-gtag/#outboundlink-component ;)

File: src/styles/_modal.scss (hunk 1 above)
Comment (blacroix) [reply src/styles/_modal.scss]: What if markup changes and comment isn't updated? Markup is already in tsx file 🤔

**Review by blacroix — CHANGES_REQUESTED** [review]

File: src/styles/_modal.scss (hunk 1 above)
Comment (avine) [reply src/styles/_modal.scss]: done

File: src/pages/about.tsx (hunk 2 above)
Comment (blacroix) [reply src/pages/about.tsx:40]: As I refactor code for this particular purpose in Invitation PR I may do it later, we can merge ;)

**Review by blacroix — APPROVED** [review]

**Pull request merged**